### PR TITLE
Rename psa

### PR DESCRIPTION
**Pull request by Digambar-Precisely**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Spatial Analytics Helm Charts
+# Private Spatial APIs Helm Charts
 
 ## Motivation
 
 1. **Flexibility of deployment**
 
-   Spatial services are delivered as separate microservices in multiple Kubernetes pods using container-based delivery.
+   Private Spatial APIs services are delivered as separate microservices in multiple Kubernetes pods using container-based delivery.
```
[the remark this answers is not in the document]
Comment (Digambar-Precisely) [reply README.md:7]: Done

File: README.md [hunk 2]
```diff
@@ -29,32 +29,32 @@
 
    Can be deployed on premise or to a cloud provider. Portability and flexibility in multi-cloud environments.
 
-> This solution is specifically for users who are looking for a REST Spatial Analytics API and Kubernetes based deployments.
+> This solution is specifically for users who are looking for a REST Private Spatial APIs and Kubernetes based deployments.
```
[the remark this answers is not in the document]
Comment (Digambar-Precisely) [reply README.md:32]: Done.

File: README.md [hunk 3]
```diff
@@ -29,32 +29,32 @@
...
+The following diagram shows an overview of Private Spatial APIs 
 
-![architecture.png](images/spatial_cloud_native_architecture.png)
+![architecture.png](images/private_spatial_apis_architecture.png)
 
-Spatial Cloud Native enables an organization to rapidly integrate location information into business applications and processes. This enables organizations to create and embed maps, understand spatial relationships, and carry out spatial calculations.
+Private Spatial APIs enables an organization to rapidly integrate location information into business applications and processes. This enables organizations to create and embed maps, understand spatial relationships, and carry out spatial calculations.
 
-The deployment is available as Micro Services architecture which can be deployed into cloud native Kubernetes environments such as Microsoft Azure (AKS), Amazon Web Services (EKS) and Google Cloud Platform (GKS) and scaled as needed.
+The deployment is available as Micro Services architecture which can be deployed into Private Spatial APIs Kubernetes environments such as Microsoft Azure (AKS), Amazon Web Services (EKS) and Google Cloud Platform (GKS) and scaled as needed.
```
[the remark this answers is not in the document]
Comment (Digambar-Precisely) [reply README.md:44]: Done

File: README.md [hunk 4]
```diff
@@ -89,8 +89,8 @@ Create a  persistent volume for storing file based spatial data/various caches.
 Install a MongoDB databse or use an existing instance. Click on the link to get steps for specific cloud platform:
 [EKS](./docs/guides/eks/QuickStartEKS.md#step-5-prepare-a-database-for-repository) | [GKE](./docs/guides/gke/QuickStartGKE.md#step-5-prepare-a-database-for-repository) | [AKS](./docs/guides/aks/QuickStartAKS.md#step-5-prepare-a-database-for-repository)
 
-### 6. Install Spatial Analytics Helm Chart
-Deploy Spatial Analytics helm chart to K8s cluster. Click on the link to get steps for specific cloud platform:
+### 6. Install Private Spatial APIs Chart
```
[the remark this answers is not in the document]
Comment (Digambar-Precisely) [reply README.md:92]: Done.

File: README.md [hunk 5]
```diff
@@ -106,17 +106,17 @@ There are various utilities for:
 
 More details on Spatial Utilities can be found [here](./docs/guides/spatial-utilities.md).
 
-## Spatial Analytics Helm Version Chart
+## Private Spatial APIs Version Chart
```
[the remark this answers is not in the document]
Comment (Digambar-Precisely) [reply README.md:109]: Done

File: docs/Metrics.md [hunk 6]
```diff
@@ -2,8 +2,8 @@
 
 ## Generating Insights from Metrics
 
-The Spatial Analytics application microservices expose metrics which can be used for monitoring and troubleshooting the
-performance and behavior of the Spatial Analytics application.
+The Private Spatial APIs expose metrics which can be used for monitoring and troubleshooting the
+performance and behavior of the Private Spatial APIs application.
```
Comment (gurudutt-precisely) [inline docs/Metrics.md:6]: Remove "application"

**Review by gurudutt-precisely — APPROVED** [review]

**Pull request merged**